8.2.32 Release # DO NOT DELETE

su-sws-jira opened the pull request.

su-sws-jira committed:
8.2.32

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr823-3mmvyginrpdpeoxqedqjwfon4o56c1ak.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/603ff81bf285afdd679e4b42

Visual Diffs:
- https://dashboard.tugboat.qa/603ff81bf285afdd679e4b42/visualdiffs

Commit:
STN-623  Bug fix dark masthead pipe (#817)

* fix(STN-623): bug dark masthead pipes

* docs(STN-623): lockup style documentation

* chore: compiled css

Co-authored-by: Kasey Bonifacio <kasey@heysparkbox.com>

Commit:
STN-691: Prevent Bullet List Numbers from Overlapping when there is Multiple Digits (#820)

* fix (STN-691): Bullet list numbers overlap with text when multiple digits

* feat: add compiled CSS

Commit:
STN-688 : Refine Captions/Credits for Vertical Linked Card (#819)

* feat (STN-688): Refine Captions/Credits for Vertical Linked Card

Increase spacing to allow all caption/credit content to display

* feat: restrict padding updates for vertical linked cards to Colorful only at screens larger than mobile only

* feat: update caption/credit positioning media image field list to also target postcard images

* feat: add compiled CSS

Commit:
feat (STN-694): Merge Refined Padding for hb-raised-cards on Colorful (#824)

* feat (STN-694): Merge Refined Padding for hb-raised-cards on Colorful

* feat: add compiled CSS

Commit:
Fix: Adjust caption credit sizing and padding on small screens (#826)

* fix: update vertical card caption on sm and xs screens

* chore: compile css

Commit:
STN-661: Carousel Height Refinement (#812)

* feat (STN-661): Carousel Height Refinement

* feat: parse through JS to find the heights of the boxes which contain content inside of carousels

* feat: add JS timeout functions to prevent the window event listener from firing the restrictHeight function until the window is no longer being resized

* feat: add hb-carousel-reset-height utility class to docs

* feat: prevent window resize event listener from listening on pages that do not have carousel slides

* feat: begin implementing review feedback

- add variable for medium screen breakpoint to describe the number we are targeting
- remove unnecessary number from parseInt function
- refactor when the restrictHeight function runs to prevent it from running at smaller screen sizes

* feat: refactor 2 loops into 1 loop to improve performance

* feat: remove boxHeightArray variable declaration from within the for loop to prevent it from being redeclared each time the code runs

* fix: update boxHeightArray variable declaration to allow a baseline value to be set for each individual carousel on a page

* feat: refactor for loops into forEach loops

* fix: hb-carousel-reset-height utility class to function correctly

* feat: add compiled CSS & JS

Commit:
STN-580  Vertical Date stacked cards icons (#828)

* feat(STN-580): Add icons to vertical stacked cards

Add the class as a theme setting and update theme setting text

pookmish committed:
HSD8-1008 add campus image gallery (#829)

* HSD8-1008 add campus image gallery

* Exclude gallery for now

* changed image style

* CC fix

* Added very minimal styles

pookmish committed:
Sparkbox Sprint 35

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr823-3mmvyginrpdpeoxqedqjwfon4o56c1ak.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/603ff81bf285afdd679e4b42

Visual Diffs:
- https://dashboard.tugboat.qa/603ff81bf285afdd679e4b42/visualdiffs

pookmish committed:
HSD8-1002 Add url validation on links in the cap importer

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr823-3mmvyginrpdpeoxqedqjwfon4o56c1ak.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/603ff81bf285afdd679e4b42

Visual Diffs:
- https://dashboard.tugboat.qa/603ff81bf285afdd679e4b42/visualdiffs

joegl committed:
HSD8-1011: Added iranianstudies to multisite. (#831)

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr823-3mmvyginrpdpeoxqedqjwfon4o56c1ak.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/603ff81bf285afdd679e4b42

Visual Diffs:
- https://dashboard.tugboat.qa/603ff81bf285afdd679e4b42/visualdiffs

pookmish committed:
HSD8-1015 Add views data export module

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr823-3mmvyginrpdpeoxqedqjwfon4o56c1ak.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/603ff81bf285afdd679e4b42

Visual Diffs:
- https://dashboard.tugboat.qa/603ff81bf285afdd679e4b42/visualdiffs
